### PR TITLE
fix punctuation, tense, no issue

### PR DESCRIPTION
**Pull request by almen-ng**

### CODE REVIEW

File: clusters/release_notes/known_issues.adoc [hunk 1]
```diff
@@ -67,7 +67,7 @@ After you delete a `ManagedClusterSet`, the label that is added to each managed
 
 If you create a Hive `ClusterClaim` against a `ClusterPool` and manually set the `ClusterClaimspec` lifetime field to an invalid golang time value, the product stops fulfilling and reconciling all `ClusterClaims`, not just the malformed claim.  
 
-If this error occurs. you see the following content in the `clusterclaim-controller` pod logs, which is a specific example with the pool name and invalid lifetime included:
+If this error occurs, you will see the following content in the `clusterclaim-controller` pod logs, which is a specific example with the pool name and invalid lifetime included:
```
Comment (swopebe) [inline clusters/release_notes/known_issues.adoc]: This whole line needs to change. We try not to use will bc we want present tense, but "If this error occurs" is not concise and unnecessary.

Comment (swopebe) [reply clusters/release_notes/known_issues.adoc]: @almen-ng changing to:

You see the following error in the `clusterclaim-controller` pod logs, which is a specific example with the `PoolName` and invalid `lifetime` included:

Comment (swopebe) [reply clusters/release_notes/known_issues.adoc]: These changes are to keep within standards of IBM/RH Style manual: conciseness, tense, and monospace code elements. TY for helping make this better.

File: clusters/release_notes/known_issues.adoc [hunk 2]
```diff
@@ -67,7 +67,7 @@ After you delete a `ManagedClusterSet`, the label that is added to each managed
 
 If you create a Hive `ClusterClaim` against a `ClusterPool` and manually set the `ClusterClaimspec` lifetime field to an invalid golang time value, the product stops fulfilling and reconciling all `ClusterClaims`, not just the malformed claim.  
 
-If this error occurs. you see the following content in the `clusterclaim-controller` pod logs, which is a specific example with the pool name and invalid lifetime included:
+You see the following error in the `clusterclaim-controller` pod logs, which is a specific example with the `PoolName` and invalid `lifetime` included:
```
Comment (jc-berger) [inline clusters/release_notes/known_issues.adoc:70]: Agree with @swopebe with her revisions. Feel free to commit Brandi's changes:

```suggestion
You see the following error in the `clusterclaim-controller` pod logs, which is a specific example with the `PoolName` and invalid `lifetime` included:
```

**Review by jc-berger — COMMENTED** [review]
Please implement Brandi's changes. Thanks!

File: clusters/release_notes/known_issues.adoc (hunk 2 above)
Comment (almen-ng) [reply clusters/release_notes/known_issues.adoc:70]: lgtm!

**Pull request merged**